src/content/en/updates/2019/06/image-support-for-async-clipboard.md

```diff
@@ -0,0 +1,299 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images.
+
+{# wf_published_on: 2019-06-30 #}
+{# wf_updated_on: 2019-06-30 #}
+{# wf_featured_image: /web/updates/images/generic/photo.png #}
+{# wf_tags: chrome76,cutandcopy,execcommand,input,clipboard #}
+{# wf_featured_snippet: The Async Clipboard API, the text-focused portion of which we’ve shipped in Chrome 66, has learned a new trick. As of Chrome 76, it now also handles images. #}
+{# wf_blink_components: Platform>Apps>AppLauncher>Install #}
+
+# Image Support for the Async Clipboard API {: .page-title }
+
+{% include "web/_shared/contributors/thomassteiner.html" %}
+
+In Chrome&nbsp;66, we shipped the [text-focused portion](/web/updates/2018/03/clipboardapi)
+of the Asynchronous [Clipboard API](https://w3c.github.io/clipboard-apis/)
+that replaced the previous method that was based on
```

Comment:
This first sentence is really long. Could we split it up or shorten it? Maybe cut out "the previous method that was based on"

Reply:
I cut out the suggested part.